packages/stable/src/api/models/codegen.skip.json

```diff
@@ -0,0 +1,10 @@
+{
+  "_": "Due to issue with codegen duplicating types, we skip this for now.",
```

Comment:
Can you expand on this issue?

Reply:
Sure! you can replicate this by renaming it from 'codegen.skip.json' to 'codegen.json' and running `yarn codegen --package=stable generate-types`

The `exports.gen.ts` tries to export these types, but they don't exist inside of the `types.gen` file.
It seems like the code generation is correctly identifying 'shared types', but theyre not correctly being removed from their respective types.gen file and into the common one. e.g. in this case, these types exist in `api/datamodels/types.gen.ts`, when they should be moved into `types.gen`
<img width="1226" alt="image" src="https://github.com/cognitedata/cognite-sdk-js/assets/38425820/5676bd45-eefe-40a0-9cef-b17f48d23211">

The code generation portion would take someone unfamiliar with the code a while to dig into. I tried, it was a bit tricky to follow. so we decided to generate the types once.

does that make sense?

Reply:
I think that is good way to get around that issue. I'm all for the tool simply helping you to reduce time and typos, otherwise the value is lost.

Ofcourse it should be fixed, but I'll need to get back to this a bit later.